### ISSUE TITLE
Potential MySQL Injection

### ISSUE THREAD

**Issue by egold555** [egold555/PVPClientHTTPServer#1]
```database.query("SELECT * FROM `usermap` WHERE uuid = '" + uuid + "'", function (err, result, fields) {
        //code here
 }```

Is uuid sanitized?

**Comment by cfpwastaken** [comment]
yep MySQL injection. Is unlikely to happen because UUIDs are provided by Mojang.
